README.md

```diff
@@ -414,24 +414,9 @@ CPMAddPackage(
 
 ### [Boost](https://github.com/boostorg/boost)
 
-Boost is a large project and will take a while to download. Using
-`CPM_SOURCE_CACHE` is strongly recommended. Cloning moves much more
-data than a source archive, so this sample will use a compressed
-source archive (tar.xz) release from Boost's github page.
```

Comment:
This removed bit is a very important part of this example. It coaches the user in best practices for packaging.

Reply:
You are right. I'll try to save these lines of notes.

Reply:
I guess if add them to `## Tips` that would be ok